manifests/plugin/dynflow/params.pp

```diff
@@ -0,0 +1,8 @@
+# Default parameters for the Dynflow smart proxy plugin
+class foreman_proxy::plugin::dynflow::params {
+  include ::foreman_proxy::params
+
+  $enabled           = true
+  $listen_on         = 'https'
+  $group             = $::foreman_proxy::params::user
```

Comment:
Or just remove it as an option exposed to the user? Rex doesn't much care about the ownership of the proxy file, if that ever changes we can put it back.